questdb-partitioned/create.sql

```diff
@@ -1,73 +1,73 @@
 CREATE TABLE hits
 (
     WatchID long,
-    JavaEnable short,
-    Title string,
-    GoodEvent short,
+    JavaEnable byte,
```

Comment:
sorry for an obvious question, but is `byte` big enough to represent all the possible values?

Reply:
Yes, this and many other columns are boolean-like, i.e. they take only 0 and 1 as their values. I've checked distinct value for all columns before changing their type.